scss/_patterns_footer.scss

```diff
@@ -1,98 +1,47 @@
-// scss-lint:disable QualifyingElement, IdSelector
+// Footer pattern styling
+@mixin vf-p-footer {
 
-// Default footer styling
-@mixin vf-footer {
-  body footer.global #nav-global li:first-of-type a {
-    margin-left: 0;
+  .p-footer {
+    font-size: .75rem;
   }
 
-  footer.global {
-    background: none;
-    border-top: 0;
-    box-shadow: inset 0 2px 2px -1px $color-mid-grey;
-    clear: both;
-    display: block;
-    padding: 30px 10px 20px;
-    position: relative;
-    width: 100%;
+  .p-footer-links {
```

Comment:
I feel all pattern styling should be wrapped under its main pattern. In this case `.p-footer`.